services/src/layers/layer.rs

```diff
@@ -254,6 +272,18 @@ pub struct AddLayerCollection {
     pub properties: Vec<Property>,
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone, ToSchema)]
+#[serde(rename_all = "camelCase")]
+// TODO: validate user input
```

Comment:
check it

Reply:
131db49ea68b62b3df27d14f617301b6917e3e2a